spec/controllers/api/v1/notes_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+describe Api::V1::NotesController, type: :controller do
+  describe 'GET #index' do
+    let(:user) { create(:user) }
+    let(:user_notes) { create_list(:note, 5, user: user) }
+
+    let!(:expected) do
+      ActiveModel::Serializer::CollectionSerializer.new(notes_expected,
+                                                        serializer: IndexNoteSerializer).to_json
+    end
+
+    context 'when fetching all the notes for user' do
+      let(:notes_expected) { user_notes.sort_by(&:id).reverse }
+
+      before { get :index }
+
+      it 'responds with the expected notes json' do
+        expect(response_body.to_json).to eq(expected)
+      end
+
+      it 'responds with 200 status' do
+        expect(response).to have_http_status(:ok)
+      end
+    end
+
+    context 'when fetching notes with page and page size params' do
+      let(:page)            { 1 }
```

Comment:
```suggestion
      let(:page) { 1 }
```
generalmente, por la config de nuestros proyectos, el linter te pide dejar un espacio

Reply:
Ah, qué raro, no me lo pidió. No me di cuenta que había quedado así, lo había sacado del otro controller. Ahora lo corrijo.